docker/dockerfiles/polkadot-parachain/polkadot-parachain_builder.Containerfile

```diff
@@ -1,4 +1,4 @@
-# This file is sourced from https://github.com/paritytech/polkadot/blob/master/docker/dockerfiles/polkadot/polkadot_builder.Dockerfile
+# This file is sourced from https://github.com/paritytech/polkadot-sdk/blob/master/docker/dockerfiles/polkadot/polkadot_builder.Dockerfile
```

Comment:
The issue is not introduced by your PR but would you mind renaming that file to `.Dockerfile` ?